.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 transpiled
 /.idea/
+/coverage/
```

Comment:
Added ability to run `npm test -- --coverage` to see how much we still need to test. This generates a directory that needs to be ignored in git.